src/PowerShellTips/2023-10-02-quickly-create-tmporary-file-in-powershell.ps1

```diff
@@ -0,0 +1,29 @@
+$tip = [tiPS.PowerShellTip]::new()
+$tip.CreatedDate = [DateTime]::Parse('2023-10-02')
+$tip.Title = 'Quickly create tmporary file in PowerShell'
+$tip.TipText = @'
+Quickly create a temporary file using dotnet class.
+
+This is handy when you want save data to a temporary log file and not worry about path/permissions/filename.
+Temporary file will be created in "C:\Users\USER\AppData\Local\Temp\1\some-tmp.tmp" and automatically cleaned up after reboot.
+'@
+$tip.Example = @'
+$tmpFile = [System.IO.Path]::GetTempFileName()
+# Easily write/read/remove
+"Some data to be saved" | Out-File $tmpFile
+# Cleanup when you no longer need it
+Remove-Item $tmpFile
+'@
+$tip.Urls = @(
+	'https://learn.microsoft.com/en-us/dotnet/api/system.io.path.gettempfilename?view=net-7.0#system-io-path-gettempfilename'
```

Comment:
Great tip! I did a quick Google to see if there was a PowerShell native way to do it and found this:

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/new-temporaryfile

You may want to mention both methods in the example code, or change the example code to just show the PowerShell native way.